crates/daphne/src/roles/mod.rs

```diff
@@ -528,14 +526,13 @@ mod test {
                 .unwrap()
         }
 
-        pub fn leader_req<M: ParameterizedEncode<DapVersion>>(
+        pub fn leader_req<M>(
             task_id: &TaskId,
             task_config: &DapTaskConfig,
             agg_job_id: Option<&AggregationJobId>,
             media_type: DapMediaType,
-            msg: M,
-        ) -> DapRequest {
-            let payload = msg.get_encoded_with_param(&task_config.version).unwrap();
+            payload: M,
```

Comment:
"M" -> "P" so that we use the same generic parameter name as the definition of `DapRequest`